backend-services/src/data_service/schema/actions.py

```diff
@@ -69,3 +69,31 @@ class DeleteOtpRecipientTrigger(BaseModel):
     @classmethod
     def valid_object_id_hex_representation(cls: type, v: str) -> str:
         return valid_hex_representation(cls, v)
+
+
+class CreateRecurringPayment(BaseModel):
+    """
+    Recurring payment creation paramaters.
```

Comment:
Perhaps, the MongoDB model should also be saving the phone number used to send a SMS notification to the user about a successful (or failed) payment? This is assuming that this task falls under the job server.

Reply:
I don't think any plans to notify users of successful/failed payment via SMS was included in this implementation.